src/bun.js/webcore/response.zig

```diff
@@ -798,6 +798,7 @@ pub const Fetch = struct {
             this.request_headers = Headers{ .allocator = undefined };
 
             if (this.http != null) {
+                allocator.free(this.http.?.client.unix_socket_path);
```

Comment:
this could be inside of HttpClient deinit? who owns the memory? should we set unix_socket_path to empty string after this and put it inside clearData?